apps/omg_eth/test/fixtures.exs

```diff
@@ -72,7 +72,8 @@ defmodule OMG.Eth.Fixtures do
 
   defp call_contract(contract, signature, args, return_types) do
     data = ABI.encode(signature, args)
-    {:ok, return} = Ethereumex.HttpClient.eth_call(%{to: contract, data: Encoding.to_hex(data)})
+    from = Application.fetch_env!(:omg_eth, :eth_call_from_address)
```

Comment:
you call everywhere `Application.fetch_env!(:omg_eth, :eth_call_from_address)`

Can you create a wrapper, for example, in `OMG.Configuration` for this function?